yoda-toolfile.spec

```diff
@@ -15,7 +15,9 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/yoda.xml
     <environment name="LIBDIR" default="$YODA_BASE/lib"/>
     <environment name="INCLUDE" default="$YODA_BASE/include"/>
   </client>
-  <use name="boost"/>
+  <use name="python"/>
+  <use name="cython"/>
+  <use name="root_cxxdefaults"/>
```

Comment:
Could you share the build log?
Could you check all executables to make sure that none are dynamically linked to ROOT? I think, we don't ship static ROOT libs, but you should check.
Maybe it just records ROOT location for future use?

Please, investigate.